### PR TITLE
fix issue

### PR DESCRIPTION
**Pull request by badb**
https://github.com/imapp-pl/golem/issues/140

### CODE REVIEW

**Comment by badb** [comment]
This value is only used as a string, so it's also possible to make a fix in tests and not in function code. But we may need to want to use number in the future so changing a function may be a better solution.

File: gnr/customizers/configurationdialogcustomizer.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def load_data(self):
     @staticmethod
     def du(path):
         try:
-            return subprocess.check_output(['du', '-sh', path]).split()[0]
+            return subprocess.check_output(['du', '-sh', path]).split()[0].replace(",", ".")
```
Comment (chfast) [inline gnr/customizers/configurationdialogcustomizer.py]: Maybe it's better to skip `-h` option.

Comment (chfast) [inline gnr/customizers/configurationdialogcustomizer.py]: Is it faster than pure Python? E.g. http://stackoverflow.com/questions/1392413/calculating-a-directory-size-using-python

**Comment by chfast** [comment]
Ok. However, consider adding test for `get_dir_size()`: https://codecov.io/github/imapp-pl/golem/compare/017d052...47be47e

**Comment by azawlocki** [comment]
Fine for me but let's try to meet this 80% codecov target.

**Pull request merged**